tests/test_yokogawa/test_yokogawa_6370.py

```diff
@@ -31,6 +29,7 @@
 
 def test_channel_is_channel_class():
     inst = ik.yokogawa.Yokogawa6370.open_test()
+    assert inst._channel_count == len(inst.Traces)
```

Comment:
`ik.yokogawa.Yokogawa6370.Traces` is 5 channels long, this is provided to the proxy list. Making sure here we set this now, as it is a new variable defined in the metaclass.